Tim Zulf: WIP

Tulf opened the pull request:
### **1. Task - Style and add to the form**

- [x]  Add an email input
- [x]  Add a 3rd option in the radio buttons and name it "island"
- [x] Style the form by wrapping each input and all the radio buttons in a form-group

### **2. Task - Build Utility classes into the site**

- [x] Use the flex utility to vertically center and move the navigation to the far right of it's container.
- [x] Use the border radius utility to round the corners on the images named adventure.jpg and fun.jpg;
- [x] Use the the display utility sticky-top on the the container-fluid div.
- [x] Use the spacer utility to give enough top-margin or bottom-margin to any element that looks like they are too close together. Remember the syntax looks like: mt-1

### **3. Task - Build JavaScript Components**

**_Help the project managers out by making html comments where you used your JavaScript Components <!-- JavaScript Component: Carousel --> would be a great example to let them know where you put it._**

- [x] Introduce a carousel somewhere on the site showcasing the images found in the img folder. It's up to you where it goes!

**_You must use at least 2 of the following JavaScript components listed here:_**

- [ ] Collapse
- [x] Dropdowns
- [x] Modals
- [x] Tooltips
- [ ] Scrollspy
- [ ] Navbar

**_If you need to add content for the new components (like collapse for example) just use lorem ipsum for your text!_**


- [ ] **Stretch Task - Add validation to your form and make every field required.**

Tulf committed:
Tim Zulf: WIP

Added email form: frist commit

groov1234 commented:
Hey Tim! Nice work! I see you've finished a Task. It's great that you're breaking the project down. Try making a commit with each check marked completion.

Tulf committed:
added radio buttons, form changed to form-group

Tulf committed:
Task2: Complete

- Added margin tops to all smushed rows: was not able to change with margin bottom
- Spaced Nav bar with flex utility
- added border-radius, changed with custom styling
- fixed sticky nav bar to top of page"

Tulf committed:
attention to detail bb manma didn't raise no slouch

Tulf committed:
Task 3 Complete:

Added:
Carousel
Dropdown
Tooltip
Modal
I appreciate you giving me equally sized imaged xD
